packages/perseus/src/styles/widgets/orderer.css

```
@@ -1,10 +1,22 @@
 .orderer {
     position: relative;
+    /* Fix for wrapping issue caused by CSS reset removal */
+    box-sizing: border-box;
+    margin: 0;
+    padding: 0;
+    width: 100%;
```

It's hard to know what's for the fix and what isn't. Would it be useful to mark the end?

```suggestion
    /* Start fix for wrapping issue caused by CSS reset removal */
    box-sizing: border-box;
    margin: 0;
    padding: 0;
    width: 100%;
    /* End fix */
```